staging/src/k8s.io/client-go/rest/request.go

```diff
@@ -519,14 +519,17 @@ func (r Request) finalURLTemplate() url.URL {
 		newParams[k] = v
 	}
 	r.params = newParams
-	url := r.URL()
+	u := r.URL()
+	if u == nil {
```

Comment:
this can not happen, right? it always return an empty URL , it never returns nil

> // URL returns the current working URL.
> func (r *Request) URL() *url.URL {

Reply:
Correct it can't, but otherwise there is no way to ignore the linter AFAIK. Do you have a suggestion for this? :thinking:

Reply:
ah, lol, didn't got that detail

Reply:
I pasted the error without looking into it :upside_down_face: 
/lgtm

Reply:
I wonder if the linter guess that the pointer might be nil based on this check:
> if url != nil && r.c.base != nil && strings.Contains(url.Path, r.c.base.Path) {

Since url can't be nil, what would happened if you were to remove the check is, would the linter still complain?

Reply:
> I wonder if the linter guesses that the pointer might be nil based on this check:
> 
> > if url != nil && r.c.base != nil && strings.Contains(url.Path, r.c.base.Path) {
> 
> Since url can't be nil, what would happen if you were to remove the check, would the linter still complain?

No, I think it simply knows that `url` is a pointer and we are accessing the pointer without checking if it's not nil.